### PR TITLE
modified import process to:

### PR DESCRIPTION
**Pull request by aricart**
- enable user to pick from locally available exports
- if picked export is private, attempt to generate activation

### CODE REVIEW

File: cmd/addaccount_test.go [hunk 1]
```diff
@@ -126,23 +126,30 @@ func Test_AddAccountInteractiveSigningKey(t *testing.T) {
 	defer ts.Done(t)
 
 	s1, pk1, _ := CreateOperatorKey(t)
-	_, pk2, _ := CreateOperatorKey(t)
 
-	_, _, err := ExecuteCmd(createEditOperatorCmd(), "--sk", pk1, "--sk", pk2)
+	t.Log("pk1", pk1)
```
Comment (derekcollison) [inline cmd/addaccount_test.go]: You still want this here?

File: cmd/addaccount_test.go [hunk 2]
```diff
@@ -126,23 +126,30 @@ func Test_AddAccountInteractiveSigningKey(t *testing.T) {
...
-
 	d, err := ts.Store.Read(store.JwtName("O"))
 	require.NoError(t, err)
 	oc, err := jwt.DecodeOperatorClaims(string(d))
 	require.NoError(t, err)
+
+	ac, err := ts.Store.ReadAccountClaim("A")
+	require.NoError(t, err)
+
+	t.Log(NewOperatorDescriber(*oc).Describe())
```
Comment (derekcollison) [inline cmd/addaccount_test.go]: Same here

File: cmd/addimport.go [hunk 3]
```diff
@@ -121,12 +121,147 @@ func (p *AddImportParams) SetDefaults(ctx ActionCtx) error {
...
+	}
+
+	ac, err := ctx.StoreCtx().Store.ReadAccountClaim(ctx.StoreCtx().Account.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	var filtered []AccountExport
+	for _, f := range found {
+		// FIXME: filtering on the target account, should eliminate exports the account already has
```
Comment (derekcollison) [inline cmd/addimport.go]: Not following this, what exactly are you filtering?

Comment (aricart) [reply cmd/addimport.go]: imports that match the exports they have

File: cmd/addaccount_test.go (hunk 2 above)
Comment (aricart) [reply cmd/addaccount_test.go]: thanks

File: cmd/addaccount_test.go (hunk 1 above)
Comment (aricart) [reply cmd/addaccount_test.go]: not really

File: cmd/addimport.go (hunk 3 above)
Comment (derekcollison) [reply cmd/addimport.go]: Should we really filter that? Not sure the answer is obvious TBH.

**Pull request merged**